src/app/modules/core/changepassword/changepassword.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
change password etc are all different pages. they should be in modules 

i.e app/modules/changepassword like this not in core